### DISCUSSION TITLE
Repo fork

### DISCUSSION THREAD

**Discussion by SpeedLight-Dev** [thatjar/1bot#10] [General]
Hello, I was wondering if I could use this repo for my server's bot. Thanks 💯

**Comment by thatjar** [comment]
For one server? I'm fine with clones being in only one server.

But still make sure you don't edit license/copyright notices tho, that'll be a violation of the license.

**Reply by SpeedLight-Dev** [reply]
> For one server? I'm fine with clones being in only one server.
> 
> But still make sure you don't edit license/copyright notices tho, that'll be a violation of the license.

Yes, it will be the bot of my community.

**Reply by thatjar** [reply]
Go ahead then :)
But good luck setting it up lol, I don't have any self-hosting instructions or anything like that

**Reply by SpeedLight-Dev** [reply]
> Go ahead then :) But good luck setting it up lol, I don't have any self-hosting instructions or anything like that

I just want to take some commands, not all the bot 😅

**Reply by thatjar** [reply]
ah, mb
yeah you can, go ahead (the license won't really matter here)

**Reply by SpeedLight-Dev** [reply]
> ah, mb yeah you can, go ahead (the license won't really matter here)

No worries mate! Thank you!
